spikingjelly/activation_based/ann2snn/utils.py

```
@@ -1,9 +1,48 @@
+
+    * **中文**
+
+    从指定 URL 下载文件并保存到目标路径。支持断点续传。
+
+    :param url: 文件的下载链接
+    :type url: str
+
+    :param dst: 保存文件的目标路径
+    :type dst: str
+
+    :return: 文件的总大小（以字节为单位）
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)